src/main/resources/db-scripts/migration.sql

```diff
@@ -65,7 +65,6 @@ ALTER TABLE `protein_array_target` MODIFY COLUMN `ENTREZ_GENE_ID` int(11) NOT NU
 ALTER TABLE `sanger_cancer_census` MODIFY COLUMN `ENTREZ_GENE_ID` int(11) NOT NULL;
 ALTER TABLE `gistic_to_gene` MODIFY COLUMN `ENTREZ_GENE_ID` int(11) NOT NULL;
 ALTER TABLE `cna_event` MODIFY COLUMN `ENTREZ_GENE_ID` int(11) NOT NULL;
-ALTER TABLE `cosmic_mutation` MODIFY COLUMN `ENTREZ_GENE_ID` int(11) NOT NULL;
```

Comment:
@onursumer This change in `migration.sql` looks strange to me, I think we should only adding new lines to the end of the file. e.g.
```sql
DROP TABLE IF EXISTS `cosmic_mutation`;
```